deploy/kubernetes/v1.19/storage-class.yml

```diff
@@ -1,14 +1,18 @@
+---
 apiVersion: storage.k8s.io/v1
 kind: StorageClass
 metadata:
   name: synology-iscsi-storage
-  # annotations:
-  #   storageclass.kubernetes.io/is-default-class: "true"
+  annotations:
+    storageclass.kubernetes.io/is-default-class: "true"
 provisioner: csi.san.synology.com
 # if all params are empty, synology CSI will choose an available location to create volume
 # parameters:
 #   dsm: '1.1.1.1'
 #   location: '/volume1'
 #   fsType: 'ext4'
+parameters:
+  fsType: ext4
+  location: /volume1
```

Comment:
Well, it didn't work for me without this.

Reply:
I couldn't reproduce this. Could you provide the detailed error logs when you use a storage class without these parameters?
And what is your K8s and K8s version?

Reply:
I'm using K3s 1.22.5. As for the logs and the updates to this PR, I don't have the time to work on this right now.

Reply:
I checked again - the issue persists. In particular, I need to configure `parameters.fsType=ext4` or otherwise some containers would crash with a "permission denied" error on startup. The weird part is that this happens with some selected container images only (e.g. prom/prometheus:v2.19.1 and grafana/grafana:7.0.3) but not others. The behaviour is consistent, however. This may be some strange interaction between these containers and the csi-plugin container.